index.js

```diff
@@ -1,5 +1,99 @@
-console.log("hola");
+"use strict";
+const listOfCards = document.querySelector("#lista");
+const shuffle = document.querySelector(".shuffle");
+const topCard = document.querySelector("#top-card");
 
-const mainContainer = document.querySelector("#app");
+//GENERANDO ARRAY CON CARTAS
 
-mainContainer.innerText = "Hello World";
+const cardGeneration = function () {
+  const cardsArrGen = [];
+  const clubs = ["clubs", "diamonds", "hearts", "spades"];
+  const cards = ["As", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K"];
+
+  for (let i = 0; i < 4; i++) {
+    for (let j = 0; j < 13; j++) {
```

Comment:
Same as my previous comment, no magic numbers/values, we can replace this with the length of the array of card numbers
```suggestion
    for (let j = 0; j < cards.length; j++) {
```